src/var/getDefaultOptions.js

```diff
@@ -1,9 +1,10 @@
 define([
-    'jquery'
+    'jquery',
+    'var/emojioneSupportMode'
 ],
-function($) {
+function($, emojioneSupportMode) {
     return function () {
-        return $.fn.emojioneArea && $.fn.emojioneArea.defaults ? $.fn.emojioneArea.defaults : {
+        var defaultOptions = {
```

Comment:
I think we don't need to init `defaultOptions` when `$.fn.emojioneArea.defaults` set, because it will not used anyway